Snakefile

```diff
@@ -0,0 +1,37 @@
+configfile: "1_fetch/fetch_config.yaml"
```

Comment:
if you do this, I would read the Snakemake docs about modularization (https://snakemake.readthedocs.io/en/stable/snakefiles/modularization.html). 

I think it would probably be simplest while also being sufficient to use the `include` funcationality where each phase's snakefile includes the relevant previous phases' snakefiles.

Reply:
Done! I also added a new `Snakefile` which is the main file containing the rule `all`.